ibcalvariotv/transmissao/equipamentos-e-programas/propresenter/index.md

```diff
@@ -10,6 +10,10 @@
 
 É um software que pode ser instalado no Windows ou Mac que tem como finalidade principal apresentar as letras dos louvores em uma tela, normalmente a saída é um projetor ou televisão.
 
+Ele é gratuito, porém sem algumas funcionalidades, além de conter uma marca d'agua ao apresentar algo.

```

Comment:
Acho que já tinha lido sobre a única mudança ser a marca dagua. Você sabe de outras?
Na verdade ele não é gratuíto, é que o uso de forma gratuíta tem essa limitação, posso aprovar assim e depois eu mudo.
Mas foi bom mesmo lembrar de escrever sobre isso.